community-membership.md

```diff
@@ -93,6 +93,27 @@ code reviews and work towards becoming an *approver* for the subproject that
 they are active in.  Acceptance of code contributions requires at least one
 approver in addition to the reviews by *members.*
 
+## Triager
+
+Triagers assist the maintainers and approvers with project management and 
+backlog organization. The specific workflows and triage requirements depend on 
+the project, and are set by the project maintainers.
+
+Defined by: [Triage permissions](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#repository-access-for-each-permission-level)
+
+Traigers are not required to make code contributions, but are encouraged to be
+active participants in project meetings, chat rooms, and other discussion forums.
+
+### Requirements
+
+- Nominated by a maintainer, with no objections from other maintainers.
```

Comment:
I'd suggest to list some formal requirements on project engagement. Perhaps attending SIG meetings and/or community meetings. Or being active in issues comments for some time. It's hard to maintain the sense of fairness without formal requirements

Reply:
Makes sense. I want to avoid defining a specific triage process, as project maintainers should have control over this. But a basic requirement of one month of participation would be reasonable, and match the time requirement for approvers.

Something like: 

> * Consistently attend meetings and interact with issues for at least 1 month.